src/doc/en/developer/portability_testing.rst

```diff
@@ -1164,23 +1164,7 @@ provides sample ``devcontainer.json`` configuration files
 <https://github.com/sagemath/sage/tree/develop/.devcontainer/>`_ for this
 purpose.
 
-To get started, symlink (or copy) one of the sample files to
-``$SAGE_ROOT/.devcontainer/devcontainer.json``. For example, choose
-`$SAGE_ROOT/.devcontainer/portability-ubuntu-jammy-standard/devcontainer.json
-<https://github.com/sagemath/sage/tree/develop/.devcontainer/portability-ubuntu-jammy-standard/devcontainer.json>`_, which uses the Docker image based on ``ubuntu-jammy-standard``,
```

Comment:
Can't remove this part completely because these instructions are specifically for the portability-.... devcontainers